content/kubernetes/release-notes/k8s-6-4-2-4.md

```diff
@@ -0,0 +1,179 @@
+
+| **Kubernetes version**  | 1.21       | 1.22       | 1.23       | 1.24       | 1.25       | 1.26       |
+|:------------------------|:----------:|:----------:|:----------:|:----------:|:----------:|:----------:|
+| Community Kubernetes    |            | deprecated | supported  | supported  | supported  | supported* |
+| Amazon EKS              |            | supported  | supported  | supported* |            |            |
+| Azure AKS               |            |            | deprecated | supported  | supported* |            |
+| Google GKE              |            | supported  | supported  | supported  | supported  |            |
+| Rancher 2.6             | deprecated | supported  | supported  | supported  |            |            |
+| Rancher 2.7             |            |            | supported* | supported* |            |            |
+| VMware TKG 1.6          |            | supported  | supported  |            |            |            |
+| **OpenShift version**   | **4.8**    | **4.9**    | **4.10**   | **4.11**   |  **4.12**  |            |
+|                         | deprecated | deprecated | supported  | supported  | supported* |            |
+| **VMware TKGI version** | **1.12**   | **1.13**   | **1.14**   | **1.15**   |            |            |
+|                         |            | deprecated | supported  | supported  |            |            |
+
+\* Support added in this release
+
+## Active-Active preview known limitations
+
+* No support for Hashicorp Vault for storing secrets (RED-95805)
+* No module support (RED-95153)
+* No support for client certificates in secrets (RED-95724)
+* No support for backup configuration (RED-95724)
+* No support for upgrading the database Redis version
+* REAADB secret status isn't updated in source cluster (RED-96296)
+
+  The workaround is to view the secret status in one of the remote clusters.
+* Invalid REAADB is not rejected by admission might get deleted after apply. (RED-96300)
+
+  Fix the problems with the REAADB and reapply. Contact support if you aren't sure why the REAADB is invalid.
+* Admission is not blocking REAADB with `shardCount` which exceeds license quota. (RED-96301)
+
+  Fix the problems with the REAADB and reapply.
+* RERC resources must have a unique name (`<rec-name>/<rec-namespace>`). (RED-96302)
+
+* Only global database options are supported, not support for specifying configuration per location.
+* Can't automatically update the cluster secret via the operator (can be updated manually)
+* No support for migration from old (manual) Active-Active database method to new Active-Active controller
+
+## Known limitations
+
+* Long cluster names cause routes to be rejected  (RED-25871)
+
+  A cluster name longer than 20 characters will result in a rejected route configuration because the host part of the domain name will exceed 63 characters. The workaround is to limit the cluster name to 20 characters or fewer.
```

Comment:
```suggestion
  A cluster name longer than 20 characters will result in a rejected route configuration because the host part of the domain name exceeds 63 characters. The workaround is to limit the cluster name to 20 characters or fewer.
```